js/ui/tree_view/ui.tree_view.search.js

```diff
@@ -58,6 +58,10 @@ const TreeViewSearch = TreeViewBase.inherit(searchBoxMixin).inherit({
         });
     },
 
+    _getRootNodeContainer: function() {
```

Comment:
Why "root"?
```suggestion
    _getNodeContainer: function() {
```

Reply:
its just that it can contain other node-containers inside, so this is like main container, but i'll rename if you dont like it)